### PR TITLE
Cypress - Add Import Account test + data-cy labels + add joe's fix on nuxt config

### PR DESCRIPTION
**Pull request by stavares843**
**What this PR does** 📖

- chore: add `data-cy` label to submit passphrase
- test: add import account test

https://user-images.githubusercontent.com/29093946/145622448-d5fc53ef-99b2-4450-a454-457fa6b3999d.mp4

- chore: adds @josephmcg 's fix on `nuxt.config.js`, there weren't any `data-cy` labels appearing locally
- test: add import test on commands to be reusable

**Special notes for reviewers** 🗒️

N/A

**Additional comments** 🎤

N/A

### CODE REVIEW

**Review by WanderingHogan — APPROVED** [review]

File: nuxt.config.js [hunk 1]
```diff
@@ -161,29 +161,11 @@ export default defineNuxtConfig({
 
   // Build Configuration: https://go.nuxtjs.dev/config-build
   build: {
-    extend(config, ctx) {
+    extend(config) {
       config.node = {
```
Comment (WanderingHogan) [inline nuxt.config.js:165]: can you add this back in, and everything inside `extend(config,ctx) {` with       `if (process.env.ENVIRONMENT === 'DEV' || 'PRODUCTION') {` and see if it lets cypress run it's tests?

**Review by WanderingHogan — CHANGES_REQUESTED** [review]

File: nuxt.config.js (hunk 1 above)
Comment (stavares843) [reply nuxt.config.js:165]: checking

Comment (stavares843) [reply nuxt.config.js:165]: tried that, labels get removed on non prod as well 💥

Comment (stavares843) [reply nuxt.config.js]: added @josephmcg 's fix, works fine 🎉

**Review by WanderingHogan — APPROVED** [review]

**Review by Jekrimo — APPROVED** [review]

**Pull request merged**